packages/framework-integration-tests/src/commands/change-cart-item.ts

```diff
@@ -18,7 +18,12 @@ import {
   before: [ChangeCartItem.beforeFn, ChangeCartItem.beforeFnV2],
 })
 export class ChangeCartItem {
-  public constructor(readonly cartId: UUID, readonly productId: UUID, readonly quantity: number) {}
+  public constructor(
+    readonly cartId: UUID,
+    readonly productId: UUID,
+    readonly quantity: number,
+    @ignore readonly test: number
+  ) {}
```

Comment:
Would the decorator also work if we define the properties at the class level, instead of in the constructor (something that is common)? I mean if we do this:
```typescript
export class ChangeCartItem {
    readonly cartId: UUID
    readonly productId: UUID
    readonly quantity: number
    @ignore readonly test: number

  // ...
```

I guess it won't work because you are using the field index in the constructor to know which field to ignore, but any of these fields are in a constructor. 
It probably works if you use the name of the field